ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -478,6 +478,9 @@ tenants:
       - name: NOT-WAN-VRF
         vrf_id: 13
         address_families: []
+        bgp:
+          # Test that we can forcefully configure BGP for this VRF even if not needed by anything.
+          enabled: true
```

Comment:
Hmmm if this works it means the comment line 477 is not true anymore right? There is no change in the intended configs

Reply:
the comment was wrong already, but I have updated it now.